x-pack/plugins/spaces/server/routes/api/v1/spaces.js

```diff
@@ -4,14 +4,48 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import Boom from 'boom';
 import { omit } from 'lodash';
+import { getClient } from '../../../../../../server/lib/get_client_shield';
 import { routePreCheckLicense } from '../../../lib/route_pre_check_license';
 import { spaceSchema } from '../../../lib/space_schema';
 import { wrapError } from '../../../lib/errors';
+import { createDuplicateContextQuery } from '../../../lib/check_duplicate_context';
 
 export function initSpacesApi(server) {
   const routePreCheckLicenseFn = routePreCheckLicense(server);
 
+  function convertSavedObjectToSpace(savedObject) {
+    return {
+      id: savedObject.id,
+      ...savedObject.attributes
+    };
+  }
+
+  const callWithInternalUser = getClient(server).callWithInternalUser;
```

Comment:
I'm assuming it's not possible to use the SavedObjectsClient here? We shouldn't be using `callWithInternalUser` at least, as this has the potential to grant unauthorized users permission to the spaces... using `callWithRequest` is at least safer, but the SavedObjectClient is preferable (unless we can't do so until the Client/Repository split actually happens).

Reply:
Gotcha, that makes sense. As long as we have it noted that we'll need to switch this in the future, LGTM